README.rst

```diff
@@ -68,9 +68,9 @@ Internationalization
 
 Please see `edx/frontend-platform's i18n module <https://edx.github.io/frontend-platform/module-Internationalization.html>`_ for documentation on internationalization.  The documentation explains how to use it, and the `How To <https://github.com/edx/frontend-i18n/blob/master/docs/how_tos/i18n.rst>`_ has more detail.
 
-.. |Build Status| image:: https://api.travis-ci.com/edx/frontend-template-application.svg?branch=master
-   :target: https://travis-ci.com/edx/frontend-template-application
-.. |Codecov| image:: https://codecov.io/gh/edx/frontend-template-application/branch/master/graph/badge.svg
-   :target: https://codecov.io/gh/edx/frontend-template-application
-.. |license| image:: https://img.shields.io/npm/l/@edx/frontend-template-application.svg
-   :target: @edx/frontend-template-application
+.. |Build Status| image:: https://api.travis-ci.com/edx/frontend-app-communications.svg?branch=master
+   :target: https://travis-ci.com/edx/frontend-app-communications
```

Comment:
I wonder if this needs to be removed from this file and in the template app? I was under the impression that we have moved away from Travis (or are moving away). Either way, I think this link 404s now?